README.md

```diff
@@ -54,9 +54,33 @@ See [contributing](./.github/CONTRIBUTING.md) guidance and the [key decisions lo
 
 This product requires **Java version 1.8** to be installed. Later versions are not supported at present.
 * Clone the repository
-* Follow the steps in the [setup instructions](./generator/README.md). The following need to be actioned for the generator, the remaining steps need to be actioned to be able to use the profiler.
-* Build the solution using your preferred method - the project is fully compatible with IntelliJ and Eclipse. You can use Maven (`mvn install -pl :generator -am`) if you prefer.
+* Follow the steps in the [setup instructions](./generator/README.md). The following need to be actioned for the generator, 
+the remaining steps need to be actioned to be able to use the profiler.
```

Comment:
could 

> The following need to be actioned for the generator, the remaining steps need to be actioned to be able to use the profiler

be reworded to be more readable? "The following steps" suggests that all below steps are for the generator, so then what are the remaining steps for the profiler?

Could it be something like:

> Run steps w-x to set up the data generator and (optionally) run steps y-z to install and set up the profiler.